b2sdk/transfer/inbound/file_metadata.py

```diff
@@ -56,6 +59,18 @@ def as_info_dict(self):
             'fileName': self.file_name,
             'contentType': self.content_type,
             'contentLength': self.content_length,
-            'contentSha1': self.content_sha1,
+            'contentSha1': self._encode_content_sha1(self.content_sha1, self.content_sha1_verified),
             'fileInfo': self.file_info,
         }
+
+    @classmethod
+    def _decode_content_sha1(cls, content_sha1):
+        if content_sha1.startswith(cls.UNVERIFIED_CHECKSUM_PREFIX):
+            return content_sha1[len(cls.UNVERIFIED_CHECKSUM_PREFIX):], False
+        return content_sha1, True
+
+    @classmethod
+    def _encode_content_sha1(cls, content_sha1, content_sha1_verified):
+        if not content_sha1_verified:
+            return '{}{}'.format(cls.UNVERIFIED_CHECKSUM_PREFIX, content_sha1)
```

Comment:
we use `%`-style formatting in this project, could you please change this (for consistency)?